docusaurus/docs-internals/editor-controls/data-controls/multiselect.md

```diff
@@ -0,0 +1,238 @@
+| `choices`         | `ChoicesSync \| ChoicesAsync`                                                                                                                                                                           |    -    | The `choices` property can be configured to either synchronously or asynchronously provide a list of selectable options for a given component. This property can take two forms: `ChoicesSync` or `ChoicesAsync`.<br/><br/>`ChoicesSync` is an array of `Choice` objects that are directly provided to the component.<br/><br/>**`Choice: { title: string; value: Literal; }`**<br/><br/> Each `Choice` object defines the following properties:<br/>`title` - A string representing the display title of the choice.<br/>`value` - A value representing the choice. This value is returned when the choice is selected.<br/><br/>`ChoicesAsync` is used to load the choices dynamically via asynchronous functions. It is an object that defines the following properties:<br/><br/>**`ChoicesAsync: { load: (value: Value, abortSignal?: AbortSignal) => Promise<Choice[]>; search: (search: string, abortSignal?: AbortSignal) => Promise<Choice[]>; }`**<br/><br/>`load` - A function that takes a value and optionally an `AbortSignal`, and returns a Promise resolving to an array of `Choice` objects. This function is used to load the initial set of choices.<br/>`search` - A function that takes a search string and optionally an `AbortSignal`, and returns a Promise resolving to an array of `Choice` objects based on the search criteria. This function is used to dynamically search and load choices based on user input.<br/> |                                                                                                                                                                                                                         |
+
+### Return value
+
+The return value of the multiselect component is an array of selected Choice objects. Each Choice object contains the title and value properties, representing the selections made by the user. This allows for easy retrieval and manipulation of the selected data in your application.
+
```

Comment:
need to object example as in slider 

![image](https://github.com/EasyBrizy/Brizy-Local-Editor/assets/52622998/c322ccdd-df91-4690-a311-40e6edb26dff)

Reply:
your return values is stringified array, so need to explain this add need to add:
```ts
{
  value: string;
}
```